File: SIMS_BACKEND/SIMS_BACKEND/Models/School.cs
namespace SIMS_BACKEND.Models
{
    public class School
    {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public string City { get; set; } = null!;
        public string Address { get; set; } = null!;
        public DateTime? UpdatedAt { get; set; }
        public DateTime? CreatedAt { get; set; }

        public ICollection<User> Users { get; set; } = null!;
        public ICollection<Equipment> Equipment { get; set; } = null!;

    }
}

File: src/sims_frontend/src/types/school.ts
import { User } from "./user"
import { Equipment } from "./equipment"

export interface School {
    id: number;
    name: string;
    city: string | null;
    address: string | null;
    updatedAt: string | null;
    createdAt: string | null;
    users: User[] | null;
    equipment: Equipment[] | null;
}
